src/containerapp/azext_containerapp/_managed_service_utils.py

```diff
@@ -33,14 +35,15 @@ def build_redis_params(resource_id, capp_name, key_vault_id):
             },
             'scope': capp_name,
         }
-        return parameters
+        return [parameters]
```

Comment:
Can I ask why here return object change to return a list?
Is it work before? Has it been tested?

Reply:
for managed kafka, there's two linker objects created when parameters for both the bootstrap server and the schema registry are provided. the list object is to have them follow the same lifecycle in create/delete. I tested it on both existing and the newly added Managed Kafka